pacu/modules/iam__enum_action_query/main.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python3
+import argparse
+import json
+from colorama import Fore, Style
+import os
+
+from pacu.core.lib import strip_lines, downloads_dir
+
+from policyuniverse.statement import Statement
+
+module_info = {
+    "name": "iam__enum_action_query",
+    "author": "Dave Yesland (@daveysec)",
+    "category": "ENUM",
+    "one_liner": "Allows you to query enumerated user and role permissions.",
+    "description": "This module allows you to query IAM permissions for users and roles and see what resources if any they have those permissions on. For example --query s3:get*,iam:create*.", # noqa
+    "services": ["IAM"],
+    "prerequisite_modules": ["iam__enum_permissions"],
+    "external_dependencies": [],
+    "arguments_to_autocomplete": [],
+}
+
+parser = argparse.ArgumentParser(add_help=False, description=module_info["description"])

```

Comment:
`add_help=False`

You don't want to add a help flag?

Reply:
Fixed